### ISSUE TITLE
help.

### ISSUE THREAD

**Issue by who15** [liangjiandeng/TDNet#2]
sorry to bother you,but i can't find the dataloader?,how to process those data confuse me!

**Comment by usamashami11** [comment]
> sorry to bother you,but i can't find the dataloader?,how to process those data confuse me!

You are right, some portions are missing from this repo, e.g. Dataloader and Metrics etc. However, you may consult this DLPAN-Toolbox since the author has used similar data preprocessing and loading methods, as used here: https://github.com/liangjiandeng/DLPan-Toolbox
Thanks

**Comment by 1327745547** [comment]
I'm a beginner, can you publish your data preprocessing code? thank you
